ACHNBrowserUI/ACHNBrowserUI/views/customTasks/CustomTaskFormView.swift

```diff
@@ -0,0 +1,89 @@
+//
```

Comment:
I wouldn't be able to save a new custom task if there is no icons provided. IMO you should add an error if the icon is let empty